share/spice/pirate_speak/pirate_speak.handlebars

```diff
@@ -1,2 +1 @@
-
-
+{{title}}
```

Comment:
We don't need a Handlebars template 👍 

Seeing as you're using the `text` template, and you've set the `title` we'll render the title for you 👍